src/Commands/QuantDrushCommands.php

```diff
@@ -70,14 +70,29 @@ private function getDrushPath() {
    * @option threads
    *   Number of threads to use (default 5)
    * @usage quant:run-queue --threads=5
+   * @option unlock
+   * Allow a queue run even if another run is in progress. (default is false)
+   * @usage quant:run-queue --unlock
```

Comment:
```suggestion
   * @option unlock
   *   Allow a queue run even if another run is in progress (default is false)
   * @usage quant:run-queue --unlock
```

Reply:
I indented and I removed the period only to be consistent with the previous parameter description (though all really should have periods)